build_tools/bazel/third_party_import/llvm-project/overlay/mlir/BUILD.bazel

```diff
@@ -2479,9 +2534,9 @@ gentbl(
     strip_include_prefix = "include",
     tbl_outs = [("-gen-llvmir-intrinsics -dialect-opclass-base LLVM_TpuOp -llvmir-intrinsics-filter int_tpu", "include/mlir/IR/llvm_tpu_ops_gen.td.inc")],
     tblgen = ":mlir-tblgen",
-    td_file = "@llvm-project//llvm:include/llvm/IR/Intrinsics.td",
+    td_file = "//third_party/llvm/llvm-project/llvm:include/llvm/IR/Intrinsics.td",
```

Comment:
So this is obviously wrong, but also it doesn't appear to be a build file we care about, so I'm inclined to submit anyway. I followed-up internally with the author of the change that added this (it's a missing rewrite rule in the TF copybara).